tutorials/firefly/OpenUniverse2024Preview_Firefly.md

```diff
@@ -39,18 +39,13 @@ OpenUniverse2024 is a project to simulate spatially overlapping imaging surveys
 - Overlapping Roman Wide-Area Survey (WAS) in the same region
 - A deep-field calibration region of the Roman WAS in the same region
 
-This data preview release consists of a subset of data from each of the five categories above. More information about data preview can be found at:
-https://irsa.ipac.caltech.edu/data/theory/openuniverse2024/overview.html
+This data preview release consists of a subset of data from each of the five categories above. More information about data preview can be found at [IRSA holding of this dataset](https://irsa.ipac.caltech.edu/data/theory/openuniverse2024/overview.html). 
 
-Firefly is an open-source web-based UI library for astronomical data archive access and visualization developed at Caltech and used by multiple space- and ground-based astrophysics archives. More information can be found here:
-https://github.com/Caltech-IPAC/firefly/blob/dev/README.md
+Firefly is an open-source web-based UI library for astronomical data archive access and visualization developed at Caltech and used by multiple space- and ground-based astrophysics archives. More information on Firefly can be found [here](https://github.com/Caltech-IPAC/firefly/blob/dev/README.md).
```

Comment:
This is absolutely nitpicking, and we don't really do copy editing anywhere, but I usually point out that linking under some descriptive text is more useful than doing it under `here` or something similarly generic.

In this particular example, I think I would add the link to the readme under the first mention of Firefly and drop the second sentence altogether.

https://heyoka.medium.com/dont-use-click-here-f32f445d1021

Reply:
That was an interesting read - thanks for sharing.

I actually thought of doing that but Vandana wrote this text so I didn't want to overdo my edits. But good to know, you feel the same - will tackle this if/when I make next PR